src/main.rs

```diff
@@ -47,12 +98,13 @@ fn main() -> Result<(), Box<dyn Error>> {
         *control_flow = ControlFlow::Poll;
 
         match event {
-            Event::WindowEvent { event, .. } =>
+            Event::WindowEvent { event, .. } => {
                 if let WindowEvent::CloseRequested = event {
                     *control_flow = ControlFlow::Exit;
                 } else {
                     input.process_event(&event);
-                },
+                }
+            }
```

Comment:
Are you sure your rustfmt didn't screw this up? This isn't how I configured it in rustfmt.toml…

Reply:
`cargo fmt` formatted it so i dunno.